nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/security/ACP.java

```diff
@@ -200,4 +200,5 @@
      */
     boolean removeACEsByUsername(String username);
 
+    String LEGACY_BEHAVIOUR_PROPERTY = "nuxeo.security.acl.legacyBehaviour";
```

Comment:
Can you put it at the top of the interface?

Reply:
Just asked to @manonlumeau and @ktouchie about behaviour. We choose to work with American vocabulary. Can you change behaviour to behavior ?